### PR TITLE
Add LazyMan IP addresses to hosts file

### PR DESCRIPTION
**Pull request by BeansIsFat**
Added lookup of current LazyMan IP address and addition to container's hosts file for the three domains used by LazyMan. See https://www.reddit.com/r/LazyMan/wiki/hostsfile for details. Based on previous code from Plex role.

LazyMan plugin for Jellyfin is available at https://github.com/crobibero/Jellyfin.Channels.LazyMan

Also fixed check of jellyfin_system_xml file in Settings Task.

### CODE REVIEW

**Comment by desimaniac** [comment]
Takout at the /dev/null redirection in [appveyor](https://github.com/Cloudbox/Community/blob/efd29e946c3041a06dff5117090c67c2c89c0197/appveyor.yml#L119) to see why CB install is failing.

**Comment by desimaniac** [comment]
Why did you close it? Leave it open until we fix dev before merging this.

**Comment by BeansIsFat** [comment]
Sorry, I misunderstood your message in Discord.

File: roles/jellyfin/tasks/main.yml [hunk 1]
```diff
@@ -116,7 +124,7 @@
 - name: Settings Tasks
   include_tasks: "jellyfin_settings.yml"
   when: not (
-        (jellyfin_system_xml.exists)
+        (jellyfin_system_xml.stat.exists)
```
Comment (desimaniac) [inline roles/jellyfin/tasks/main.yml]: Good catch with this

**Pull request merged**